README.md

```diff
@@ -1,70 +1,157 @@
+- ✔ A professional editer such as [VS Code](https://code.visualstudio.com/download).
+- ✔ An Updated web browser such as Google Chrome, you can download it from [here](https://www.google.com/chrome/).
 
-### Analyzing the Bundle Size
+### Setup
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+- Clone this [repository](git@github.com:omwomo/hello-react-front-end.git) to your desired folder:
 
-### Making a Progressive Web App
+- Run this command in your command line interface:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+```sh
+  cd [YOUR FOLDER]
+  git git@github.com:omwomo/hello-react-front-end.git
+  cd hello-react-front-end
 
-### Advanced Configuration
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+- Update the database.yml with your database credentials under the development section
+
+
+### Usage
+
+- To run the project, execute the following command:
+
+```sh
+   npm start
+```
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+**This project is deployed by the author, no permission for deployment by any other client.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Eugene Oduor**
+
+- GitHub: [@omwomo](https://github.com/omwomo)
+- LinkedIn: [@omwomo](https://www.linkedin.com/in/omwomo/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add info about each category**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/omwomo/hello-react-front-end/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, kindly drop a start ⭐️ for the [repository](git@github.com:omwomo/hello-react-front-end-.git);
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to express my heartfelt gratitude to **Microvere** for the invaluable learning experience they have provided.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
 
-### `npm run build` fails to minify
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Your license link is currently broken, please kindly fix this so that your project can be approved :+1: :sparkles: